components/filter/demo/filter-search-demo.js

```diff
@@ -96,5 +106,24 @@ class FilterSearchDemo extends LitElement {
 
 	}
 
+	_performFullManualSearch(searchValue) {
+		const keysToDisplay = [];
+		if (searchValue === '') {
+			keysToDisplay.push('admin', 'instructor');
+			this._fullData.forEach(value => {
+				if (value.selected) {
+					if (!keysToDisplay.includes(value.key)) keysToDisplay.push(value.key);
+				}
+			});
+		} else {
+			this._fullData.forEach(value => {
+				if (value.text.toLowerCase().indexOf(searchValue.toLowerCase()) > -1) {
+					keysToDisplay.push(value.key);
+				}
+			});
+		}
+		return keysToDisplay;
+	}
```

Comment:
This implements a search similar to what Wave needs. When search is empty, it will display the selected items as well as the items that are related to the header, in this case `admin` and `instructor`. When searching, it will do a regular search on the entire list